specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -51,6 +51,25 @@ directive:
      - $.definitions.AzureMetricData.properties.dimensions
      - $.definitions.TransactionEdge.properties.metadata
      - $.definitions.TransactionNode.properties.metadata
+  - suppress: AvoidAdditionalProperties
+    reason: The customProperties field is intentionally designed to allow user-defined key-value pairs for alert metadata.
+    from: AlertsManagement.json
+    where:
+     - $.definitions.customProperties
+  - suppress: ParametersInPost
+    reason: The newState parameter is required.
+    from: AlertsManagement.json
+    where:
+     - $.paths['/providers/Microsoft.AlertsManagement/alerts/{alertId}/changestate'].post.parameters
+     - $.paths['/{scope}/providers/Microsoft.AlertsManagement/alerts/{alertId}/changestate'].post.parameters
+  - suppress: XmsPageableForListCalls
+    reason: List operations follow existing AlertsManagement pagination patterns.
+    from: AlertsManagement.json
+  - suppress: GetCollectionOnlyHasValueAndNextLink
+    reason: Response models maintain compatibility with existing AlertsManagement response structures.
+    from: AlertsManagement.json
+  - suppress: MULTIPLE_API_VERSION
```

Comment:
Please don't add suppression of this before we reach an agreement on how to resolve the issue.